moped-editor/src/components/GridTable/GridTable.js

```diff
@@ -309,11 +309,6 @@ const GridTable = ({ title, query }) => {
     // The first key is the outermost nested part of the graphql query
     const section = obj[keys[0]];
 
-     // Bypass value extraction if column value should be "stringified"
-    if (query.config.columns[exp]?.stringify) {
-      return JSON.stringify(section)
-    }
-
```

Comment:
Are we going to remove the stringify feature?

Reply:
okay, i was torn about it, I removed it because we arent using it now, but if you think we might in the future, ill put it back